File: CodeForces/467a/main.py
def inp():
    return(int(input()))
def inlt():
    return(list(map(int,input().split())))
def insr():
    return(input().strip())

n = inp()
ans = 0
for i in range(n) :
    a, b = inlt()
    if b - a >= 2 :
        ans +=1
print(ans)

File: CodeForces/467a/main.cpp
#include <iostream>

int main() {

  int n;
  int ans = 0;
  std::cin >> n;

  for (int i = 0; i < n; i++) {
    int a, b;
    std::cin >> a;
    std::cin >> b;
    if (b - a >= 2) {
      ans++;
    }
  }
  std::cout << ans << std::endl;
  return 0;
}
